src/app/(auth)/login/page.tsx

```diff
@@ -41,8 +41,8 @@ export default function Login() {
       return;
     }
 
-    setEmailError(validEmail(email) ? '' : 'Invalid email.');
-    setPasswordError(password !== '' ? '' : 'Invalid password.');
+    setEmailError(validEmail(email) ? '' : 'Email not found.');
+    setPasswordError(password !== '' ? '' : 'Password is incorrect.');
```

Comment:
Could we also get rid of the periods for these error messages?

Reply:
OH also, are you still down to implement these error checks like we had mentioned the other day?